xtask/src/launcher.rs

```diff
@@ -25,18 +25,18 @@ use std::path::Path;
 use strum::IntoEnumIterator;
 use strum_macros::{Display, EnumIter};
 
-#[derive(Debug, Display, Copy, Clone, PartialEq, EnumIter)]
+#[derive(Debug, Display, Clone, PartialEq, EnumIter)]
 pub enum LauncherMode {
-    Virtual,
-    Native,
+    Virtual(String),
```

Comment:
Could you add a comment explaining the meaning of the strings?

Reply:
Done.